llm_perf/hardware/update_llm_perf_intel.py

```diff
@@ -0,0 +1,219 @@
+import os
```

Comment:
I don't think aggregating all intel backends in one file makes sense. each backend has specific optimizations that we might wanna explore independently, the file will only keep growing and will be very heterogenous.

Reply:
I modify this pr so that it is only about the pytorch backend, I will add the other backends after this one